helpers.php

```diff
@@ -0,0 +1,14 @@
+<?php 
+
+function unique_user_name($db, $username){
+    $stmt2 = $db->prepare('select name from user where name = :username');  
```

Comment:
Also, you can do a `select count(name) from user where name = ?` to get the count directly, which is less costly for the database.